code/modules/clothing/clothing.dm

```diff
@@ -178,19 +178,20 @@ BLIND     // can't see anything
 
 /obj/item/clothing/gloves/attackby(obj/item/weapon/W, mob/user, params)
 	if(istype(W, /obj/item/weapon/wirecutters))
-		if(clipped == 0)
+		if(!clipped)
 			playsound(src.loc, 'sound/items/Wirecutter.ogg', 100, 1)
-			user.visible_message("\red [user] snips the fingertips off [src].","\red You snip the fingertips off [src].")
+			user.visible_message("<span class='warning'>[user] snips the fingertips off [src].</span>","<span class='warning'>You snip the fingertips off [src].</span>")
 			clipped = 1
+			name = "mangled [name]"
+			desc = "[desc] They have had the fingertips cut off of them."
 			if("exclude" in species_restricted)
-				name = "mangled [name]"
-				desc = "[desc] They have had the fingertips cut off of them."
 				species_restricted -= "Unathi"
 				species_restricted -= "Tajaran"
+			update_icon()
 		else if(clipped == 1)

```

Comment:
Dealing with this now, going to squash commits after this one then we should be good
Thanks for pointing that out